tests/unit/amo/components/TestHome.js

```diff
@@ -80,8 +82,8 @@ describe(__filename, () => {
     const shelves = root.find(LandingAddonsCard);
     const shelf = shelves.find('.Home-FeaturedCollection').at(1);
 
-    expect(shelf).toHaveProp('header', 'Privacy matters');
-    expect(shelf).toHaveProp('footerText', 'See more privacy extensions');
+    expect(shelf).toHaveProp('header', 'Dynamic downloaders');
```

Comment:
Ideally, changing the order of shelves like this shouldn't require us to update any tests. Maybe in the future we can rewrite these so that they use `COLLECTIONS_TO_FETCH` (or some kind of config) to make all assertions.

Reply:
Yes, I've been wanting to clean this up and add some more metadata that can be used by both the file and tests, so I've gone ahead and done that. It's made for some significant changes, but it's all much better, IMO.